app/views/people/show.html.erb

```diff
@@ -1,23 +0,0 @@
-<!--SEARCHBROWSE Descriptive page name, messages and instructions-->
```

Comment:
From the "purpose" bit:
> It also removes the alphabet helper, which was used by the admin notifications feature and views/people/show.html.erb. views/people/show.html.erb is being removed here because it's not in use and was otherwise preventing the removal of the alphabet helper.